integreat_cms/cms/templates/_raw.html

```diff
@@ -9,6 +9,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, shrink-to-fit=no, user-scalable=no" />
     <title>{{ BRANDING|title }} {% trans 'Editorial System' %}</title>
     <link rel="shortcut icon" type="image/png" href="{% static 'logos/'|add:BRANDING|add:'/'|add:BRANDING|add:'-icon.svg' %}">
+    <link href='https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.css' rel='stylesheet' />
```

Comment:
Rather than loading this from a CDN, would it also be possible to include this in our webpack build (and thus the CSS minifier which drops all unnused css classes etc)...

Reply:
I've tried it in the 5. commit. I'm not sure if I understoot it correctly 😅

Reply:
Yes, exactly like this! :muscle: 
Now I ask myself whether we should outsource this into a separate webpack entrypoint to avoid the download for users who only use the map rarely... 
Maybe add a new line
```
map: "./integreat_cms/static/src/map.ts",
```
here:
https://github.com/digitalfabrik/integreat-cms/blob/15ce171cc8041bac2d24823e271632db56eaebf1/webpack.config.js#L10-L13

And import the CSS and JS used for the map in the new file...
Then we would have to add the lines
```
{% block javascript_head %}
    {% render_bundle 'map' 'js' %}
    {% render_bundle 'map' 'css' %}
{% endblock %}
```
to the poi form template.